### PR TITLE
Support for Lighting visualization server.

### PR DESCRIPTION
**Pull request by minyk**
Support for Lightning(http://lightning-viz.org/) visualization server.

### CODE REVIEW

**Comment by andypetrella** [comment]
@minyk I'd love to merge it ;). Do you have any news regarding what we miss?

**Comment by minyk** [comment]
@andypetrella At this moment, no. Maybe more readable documentation?

**Comment by andypetrella** [comment]
@minyk you mean for using it? or more centrally for the whole project 😄?

**Comment by minyk** [comment]
@andypetrella I mean just the integration. :)

**Comment by minyk** [comment]
@andypetrella I update the `readme.md` with the official repository. Just fix the sample result url after the merge. Thanks.

**Comment by andypetrella** [comment]
I feel pain about this one, it's very sad that the scala lib (https://github.com/lightning-viz/lightning-scala/issues/8) doesn't land.

I'm actually afraid this lib is not much continued, am I wrong @freeman-lab? (just wondering)

**Comment by freeman-lab** [comment]
Gah, sorry! Have had too many balls in the air for Lightning core (including a major update about to be released!), but promise I'll finish this out tonight.

**Comment by andypetrella** [comment]
Hehe don't want to put any pressure, but this is an interesting integration that I'd love to see more.

So do your tricks first man :-).

**Comment by freeman-lab** [comment]
No it's great, needed the reminder! =)

File: README.md [hunk 1]
```diff
@@ -609,6 +610,33 @@ The results will be:
...
+6. Run!
+
+The results will be:
+
+![Lightning Visualization Result](https://raw.github.com/minyk/spark-notebook/lightning-viz/images/lightning-viz.png)
+
+#### Issues
+
+* Custom CSS for `iframe` is in the `public/ipython/custom/custom.css`.
+* `lightning-scala` dose not support all functions in the `lightning`.
```
Comment (vidma) [inline README.md]: ~~dose~~ does not support

**Comment by vidma** [comment]
let's close this for now.

**Pull request closed without merging**